File: app/src/main/java/com/example/pocapplication/mainscreen/FetchListService.java
package com.example.pocapplication.mainscreen;

import com.example.pocapplication.models.Response;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface FetchListService {
  @GET("/s/2iodh4vg0eortkl/facts.json")
  Call<Response> getList();
}

File: app/src/main/java/com/example/pocapplication/mainscreen/FetchListService.kt
package com.example.pocapplication.mainscreen

import com.example.pocapplication.models.Response
import retrofit2.Call
import retrofit2.http.GET

interface FetchListService {
    @get:GET("/s/2iodh4vg0eortkl/facts.json")
    val list: Call<Response>
}